index.html

```diff
@@ -69,5 +69,181 @@ <h1>Hey There. I’m Ronald.</h1>
         </li>
       </ul>
     </main>
+    <!-- works section -->
+    <section class="works">
+      <h1 class="recent">My Recent Works</h1>
```

Comment:
- [ ] Please, It would be better for all the titles except the main that indicates the web to use another ```hx```  like ```h2, h3, h4, ...``` to replace the ```<h1>``` as we only have to use a single ```h1``` on the page title to indicate what the entire web page is about 😎😎

**Kindly fix this issue for the HTML semantic 🙏**